demisto_sdk/__main__.py

```diff
@@ -3783,7 +3785,10 @@ def pre_commit(
 
 # ====================== modeling-rules command group ====================== #
 modeling_rules_app = typer.Typer(
-    name="modeling-rules", hidden=True, no_args_is_help=True
+    name="modeling-rules",
+    hidden=True,
+    no_args_is_help=True,
```

Comment:
Why are some commands flagged with `no_args_is_help=True` or `hidden=True` while others are not? How do we decide which commands receive these flags?

Reply:
idk, it's not part of my PR
this is the docs for [`no_args_is_help`](https://typer.tiangolo.com/tutorial/commands/#show-the-help-message-if-no-command-is-given)